flexmeasures/data/models/planning/utils.py

```diff
@@ -313,8 +313,13 @@ def get_quantity_from_attribute(
         q = ur.Quantity(value)
         q = q.to(unit)
     except (UndefinedUnitError, DimensionalityError, ValueError, AssertionError):
-        current_app.logger.warning(f"Couldn't convert {value} to `{unit}`")
-        q = np.nan * ur.Quantity(unit)  # at least return result in the desired unit
+        try:
+            # Fall back to interpreting the value in the given unit
+            q = ur.Quantity(f"{value} {unit}")
```

Comment:
I guess this would only work whenever value is numeric, that is one of the following three cases:
- `float`
- `int`
- `str` that parses to a float or int

I suggest then, creating a function that checks this and if it returns true, then we interpret the value to be in the given units, else we return `np.nan * ur.Quantity(unit)`.

The function can be something as simple as:

```python
def is_numeric(value : Any) -> bool:
    try:
        float(value)
        return True
    except ValueError:
        return False
```

Reply:
Actually, the `ur.Quantity` initialization is already in a try-else block, so there doesn't seem to be a need for this.

Reply:
It was more of a style suggestion, it doesn't really avoid any issue. It's that I'm not a big fan of try...catch haha